src/platform.ts

```diff
@@ -50,6 +52,8 @@ export class EnvisalinkHomebridgePlatform implements DynamicPlatformPlugin {
     private chimeInitialized = false;
     private lastPartitionAction?: Partition;
 
+    public reportError = '\nPlease report all bugs at ' + packageJSON.bugs.url + '\n';
```

Comment:
Can we make this a constant above and follow the constant naming conventions (caps/upper case)?

Reply:
Yes, done.